### DISCUSSION TITLE
Clarifications on Sandbox Provider Role [BETA]

### DISCUSSION THREAD

**Discussion by Tim-Ganther** [catenax-eV/catenax-ev.github.io#75] [Q&A] [answered]
Hello everyone,
Catena-X introduces a new role Sandbox Provider BETA. However, when reviewing its descriptions in the Operating Model  (Jupyter version), some points remained unclear to me, which I would like to put up for discussion:
 
**1. GXDCH excluded for sandboxes**
The Service Map [Sandboxes (BETA)](https://catenax-ev.github.io/docs/next/operating-model/what-service-map#sandboxes-beta) mentions that the integration of GXDCH is excluded for sandboxes. Why is this the case? This restriction seems contradictory to me, especially with regard to German funding projects, which as R&D projects are limited to demonstrators/sandboxes.

**2. Cleansing Services of BPDM excluded for sandboxes**
The Service Map [Sandboxes (BETA)](https://catenax-ev.github.io/docs/next/operating-model/what-service-map#sandboxes-beta) excludes the Cleansing Services of BPDM for sandboxes. What is the reason behind the exclusion in sandbox environments?
 
**3. BPN usage across dataspaces / sandboxes:**
In the Service Map [Sandboxes (BETA)](https://catenax-ev.github.io/docs/next/operating-model/what-service-map#sandboxes-beta)  description, the cleansing service of the BPDM is excluded, which gives the impression that sandbox specific BPNs might be used. Isn't there the intention to issue one BPN number to a participant, that will be used for production as well as across all sandboxes (== same BPN number for all environments) as the [Sandbox Provider (BETA)](https://catenax-ev.github.io/docs/next/operating-model/what-service-map#sandboxes-beta) role description mentions the mandatory official Catena-X registration process?

Best,
Tim

**Comment by ma3u** [comment]
@Tim-Ganther: Do you refer to following chapters in the Operating Model, right?

1. Service Map -> [Sandboxes (BETA)](https://catenax-ev.github.io/docs/next/operating-model/what-service-map#sandboxes-beta)
2. Who: Roles in the Catena-X ecosystem -> [Sandbox Provider (BETA](https://catenax-ev.github.io/docs/next/operating-model/what-service-map#sandboxes-beta))

**Reply by Tim-Ganther** [reply]
Exactly!

**Reply by ma3u** [reply]
@werner-roman, @HFocken , @mihahnberlin  : Can you give feedback please?

**Reply by werner-roman** [reply]
Hello @Tim-Ganther, @ma3u,

thanks for the Feedback! Well pick it up and update the OM and provide some detailed answers!

CC: @danielmiehle

**Comment by werner-roman** [comment, marked as answer]
Created Issue with this request: [#88]
